src/main/java/uk/gov/hmcts/reform/ccd/document/am/feign/CaseDocumentBinaryDownloadClientApi.java

```diff
@@ -4,21 +4,17 @@
 import org.springframework.core.io.Resource;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestHeader;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
 
 import java.util.UUID;
 
-@FeignClient(name = "case-document-am-download-client-api",
-    url = "${case_document_am.url}"
-)
+@FeignClient(name = "case-document-am-download-client-api", url = "${case_document_am.url}")
 public interface CaseDocumentBinaryDownloadClientApi {
```

Comment:
Can rename this to CaseDocumentClientApi and merge other api into this "CaseDocumentDeleteClientApi.java"

Reply:
Funny you mention this, I had actually started doing this refactor as part of writing the tests 👍